addons/resource/models/resource.py

```diff
@@ -601,7 +601,7 @@ def interval_get_multi(self, date_and_hours_by_cal, resource=False, byday=True):
         for dt_str, hours, calendar_id in date_and_hours_by_cal:
             result = self.browse(calendar_id).schedule_hours(
                 hours,
-                day_dt=fields.Datetime.from_string(dt_str).replace(second=0),
+                day_dt=fields.Datetime.from_string(dt_str).replace( second=0, microsecond=0),
```

Comment:
the `from_string` result has no microseconds (and we should probably use the fileds methods more)